plugins/kubecarrier.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubecarrier
+spec:
+  description: KubeCarrier is an open source system for managing applications and
```

Comment:
We recommend using multi-line strings wrapped at 80 chars. See other plugins for YAML multi-line notation.